diff -ruN lp_solve_5.5.orig/lp_solve/ccc lp_solve_5.5/lp_solve/ccc
--- lp_solve_5.5.orig/lp_solve/ccc	2020-12-08 12:24:15.118953722 +0100
+++ lp_solve_5.5/lp_solve/ccc	2020-12-08 12:49:39.661428497 +0100
@@ -1,6 +1,6 @@
 :
 src='../lp_MDO.c ../shared/commonlib.c ../colamd/colamd.c ../shared/mmio.c ../shared/myblas.c ../ini.c ../fortify.c ../lp_rlp.c ../lp_crash.c ../bfp/bfp_LUSOL/lp_LUSOL.c ../bfp/bfp_LUSOL/LUSOL/lusol.c ../lp_Hash.c ../lp_lib.c ../lp_wlp.c ../lp_matrix.c ../lp_mipbb.c ../lp_MPS.c ../lp_params.c ../lp_presolve.c ../lp_price.c ../lp_pricePSE.c ../lp_report.c ../lp_scale.c ../lp_simplex.c lp_solve.c ../lp_SOS.c ../lp_utils.c ../yacc_read.c'
-c=cc
+c=$CC
 
 #determine platform (32/64 bit)
 >/tmp/platform.c
diff -ruN lp_solve_5.5.orig/lpsolve55/ccc lp_solve_5.5/lpsolve55/ccc
--- lp_solve_5.5.orig/lpsolve55/ccc	2020-12-08 12:24:15.118953722 +0100
+++ lp_solve_5.5/lpsolve55/ccc	2020-12-08 12:40:22.677752522 +0100
@@ -1,6 +1,6 @@
 :
 src='../lp_MDO.c ../shared/commonlib.c ../shared/mmio.c ../shared/myblas.c ../ini.c ../fortify.c ../colamd/colamd.c ../lp_rlp.c ../lp_crash.c ../bfp/bfp_LUSOL/lp_LUSOL.c ../bfp/bfp_LUSOL/LUSOL/lusol.c ../lp_Hash.c ../lp_lib.c ../lp_wlp.c ../lp_matrix.c ../lp_mipbb.c ../lp_MPS.c ../lp_params.c ../lp_presolve.c ../lp_price.c ../lp_pricePSE.c ../lp_report.c ../lp_scale.c ../lp_simplex.c ../lp_SOS.c ../lp_utils.c ../yacc_read.c'
-c=cc
+c=$CC
 
 #determine platform (32/64 bit)
 >/tmp/platform.c
